scripts/code_style.py

```diff
@@ -191,6 +201,11 @@ def main() -> int:
                         help='only check the specified files (default with non-option arguments)')
     parser.add_argument('operands', nargs='*', metavar='FILE',
                         help='files to check (files MUST be known to git, if none: check all)')
+    parser.add_argument('-g', '--git', action='store_true',
+                        help='allow git to use a pager for diff')
+    parser.add_argument('-p', '--pager', action='store_true',
```

Comment:
I tend to prefer flipping the default and having `--no-pager` for parity with `git diff`, but it's not a blocker.

Reply:
[Rescinded above](https://github.com/Mbed-TLS/mbedtls/pull/7819#discussion_r1253142843)

Reply:
Again, I was trying to make the default experience the same as before, just with colours.